http/src/main/scala/org/http4s/blaze/http/http2/StreamStateImpl.scala

```diff
@@ -199,7 +199,7 @@ private abstract class StreamStateImpl(session: SessionCore) extends StreamState
   final override def outboundCommand(cmd: OutboundCommand): Unit =
     session.serialExecutor.execute(new Runnable {
       def run(): Unit = cmd match {
-        case Command.Flush | Command.Connect =>
+        case Command.Flush =>
```

Comment:
This is going to conflict with your other PR, but I guess it's an obvious merge.